redis/src/commands/mod.rs

```diff
@@ -674,43 +674,43 @@ implement_commands! {
     /// Intersect multiple sorted sets and store the resulting sorted set in
     /// a new key using SUM as aggregation function.
     fn zinterstore<D: ToRedisArgs, K: ToRedisArgs>(dstkey: D, keys: &'a [K]) {
```

Comment:
here, and below `keys: &'a [K]` should be changed to `key: K`

Reply:
This is a good point for unifying the API arguments, but this might slightly affect the old behavior. I saw you added the `semver-breaking`, so I guess it's expected.